bridges/snowbridge/pallets/outbound-queue-v2/src/lib.rs

```diff
@@ -344,6 +344,7 @@ pub mod pallet {
 			let outbound_message = OutboundMessage {
 				origin,
 				nonce,
+				topic: id,
```

Comment:
Don't you need to update the `sol!` bindings and add topic to the ABI-encoded message?

Reply:
Nope, it's reconstructued from the `OutboundMessage` by the relayer

https://github.com/Snowfork/snowbridge/blob/e4af25d9a2a8cfc52245528e0ed3dbab6e046364/relayer/relays/parachain/types.go#L107

I'll update the relayer later.

Reply:
You are right, still require to update the sol bindings or the verification will fail.

Let me update and finish the local test.